tests/codestyle/all.sh

```diff
@@ -21,7 +21,16 @@ function test_illicit_function_use {
 		echo "xmlTextReaderReadString is not allowed within OpenSCAP project. Its implementation in libxml does not play well with xmlWalkerReader."
 		return 1;
 	fi
-	whitelisted=$(echo $codebase | sed 's/\S*\/source\/oscap_source.c / /g' -)
+
+	case $(uname) in
```

Comment:
How about adding `xsed` function to the `test_common.sh` and using `sed` or `gsed` there based on `uname`?